apps/api/prisma/schema.prisma

```diff
@@ -36,6 +37,7 @@ model PurchaseOrderLineItems {
   id                Int            @id @default(autoincrement())
   purchase_order    PurchaseOrders @relation(fields: [purchase_order_id], references: [id])
   purchase_order_id Int
+  item              Item   @relation(fields: [item_id], references: [id])
```

Comment:
Though the item_id was present, prisma did not know that this was a foreign key to the Item object. I added this in but it turns out, I actually did not end up using the Item property on the Purchase Order Line Item. I believe it is still good practice to add this. Let me know if y'all think otherwise.

Reply:
Prisma offer `prisma format` which helps with this. Manually adding them is fine too 👍

Reply:
Good point. Will definitely use this in the future!